.circleci/config.yml

```diff
@@ -53,7 +53,7 @@ codeception: &codeception
         name: Run Acceptance Tests
         command: |
           composer global require SU-SWS/stanford-caravan:dev-8.x-1.x
-          ~/.composer/vendor/bin/sws-caravan codeception /var/www/html --extension-dir=/var/www/test
+          ~/.composer/vendor/bin/sws-caravan codeception /var/www/html --extension-dir=/var/www/test --modules=stanford_events,stanford_events_importer,stanford_events_series
```

Comment:
Is this still necessary or did you get it figured out?

Reply:
This is combination with the cache rebuilds was the best way I could get it to pass the tests as often as possible. It still failed 1 out of 5 runs that I did.but I think the only other solution is to enable modules and create entities via the UI only.